src/security/KeyData.cc

```diff
@@ -33,7 +34,7 @@ Security::KeyData::loadX509CertFromFile()
 
     try {
         cert = Ssl::ReadCertificate(bio);
-        return true;
+        debugs(83, DBG_PARSE_NOTE(2), "Loaded signing certificate: " << *cert);
```

Comment:
The `return true` above is gone (to load intermediate certificate) but the function will return true later anyway (where GnuTLS code already returns true).